### DISCUSSION TITLE
배열선언 [버그]

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#156] [Issue Report (Legacy)]
상황 재현을 위한 스크립팅 장면
<img width="1779" alt="스크린샷 2022-01-02 오전 1 54 42" src="https://user-images.githubusercontent.com/90679926/147855717-fac82002-bece-4b18-986c-747f0b836bad.png">

인스팩터에서 크기를 할당한 채로 저장 후 종료하는 장면
<img width="778" alt="스크린샷 2022-01-02 오전 1 55 23" src="https://user-images.githubusercontent.com/90679926/147855719-2dd65a76-d211-41d7-90d5-02135cfcf198.png">

프로젝트를 재 시작하면서 배열이 사용 불가능하게 고장난 장면
<img width="778" alt="스크린샷 2022-01-02 오전 1 55 42" src="https://user-images.githubusercontent.com/90679926/147855733-9336429c-9c63-414a-8e16-e3e4aa141e48.png">

특이한 상황이긴하지만 버그로 보입니다.

**Comment by x4cf** [comment]
None이 하나라도 있으면 에디터 오류가 뜸

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 오류가 최신 버전 1.1.5 에서는 재현이 안되는데
만약 동일한 증상이 나타나면 말씀해 주세요~

**Reply by x4cf** [reply]
1.1.5버전에서 해결되었어요 :D
